src/js/script.js

```diff
@@ -1,5 +1,3 @@
-import 'babel-polyfill';
-
```

Comment:
Shouldn't you still import it?
https://github.com/babel/babel-preset-env#usebuiltins
I don't think the preset includes any of the polyfills itself.